packages/hooks/feed/useFeedPosting.ts

```diff
@@ -38,7 +43,9 @@ export const useFeedPosting = (
   const [step, setStep] = useState(
     feedPostingStep(FeedPostingStepId.UNDEFINED),
   );
-  const { balances } = useBalances(networkId, userAddress);
+
+  const selectedWallet = useSelectedWallet();
+  const { balances } = useBalances(networkId, selectedWallet?.address);
```

Comment:
are you sure about this change?

we should measure the poster (DAO in this case) funds for post fees and not the proposer funds

Reply:
> are you sure about this change?
> 
> we should measure the poster (DAO in this case) funds for post fees and not the proposer funds

Hmm, i think i did not understood this at time i modified it.
Thought it was to ensure user have money to pay the proposal, but now i understand it was checking if the dao has the fee to pay the execution of proposal.

The thing is when the error was trigger a modal showing the balance of the user popped-up instead of showing the balance of the DAO, i will fix this and revert what you highlighted to ensure DAO have enough money!

Thanks for catching this :pray: